shared/unattended/Fedora.ks

```diff
@@ -0,0 +1,48 @@
+install
+KVM_TEST_MEDIUM
+GRAPHICAL_OR_TEXT
+lang en_US
+keyboard us
+network --bootproto dhcp --hostname atest-guest
+rootpw 123456
+firewall --enabled --ssh
+selinux --enforcing
+timezone --utc America/New_York
+firstboot --disable
+bootloader --location=mbr --append="console=tty0 console=ttyS0,115200"
+zerombr
+poweroff
+KVM_TEST_LOGGING
+
+clearpart --all --initlabel
+autopart
+
+%packages --ignoremissing
+@standard
+@c-development
+@development-tools
+python
+net-tools
+sg3_utils
+%end
+
+%post
```

Comment:
For the kickstart, I'd mention the same things as I have in #2247, you can comment there so we don't copy paste back and forth the discussion.

Reply:
I'd recommend bringing it back first and then adjusting it to "new era", but not a strong preference, though.

Reply:
^^ by first I mean unchanged.

Reply:
I get your point, especially if something is broken, but I'm afraid noone will do the adjustment afterwards, you know how it works - if it ain't broke, don't fix it, so we'll end up copying the same kickstart for other use cases forever, IMO better do it once and properly.